homework-solutions/js/hw-5/task-1.js

```diff
@@ -5,6 +5,11 @@
  */
 
 let evenNumbersResult = '';
+for (let i = 10; i >= 0; i -= 2) {
+  evenNumbersResult = evenNumbersResult + `-${i}`;
+}
+evenNumbersResult = evenNumbersResult.replace('-', '');
+console.log(evenNumbersResult);
```

Comment:
Как вариант
- убрали `replace`
- сократили присвоение оператором `+=` [mdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Addition_assignment) 
```suggestion
for (let i = 10; i >= 0; i -= 2) {
  evenNumbersResult += i % 2 === 0 && i !== 10 ? `-${i}` : `${i}`;
}

console.log(evenNumbersResult);
```

Reply:
спасибо. хороший вариант